src/io/flutter/view/FlutterView.java

```diff
@@ -296,16 +287,46 @@ private void addInspectorPanel(String displayName,
       treeType != InspectorService.FlutterTreeType.widget || !inspectorService.isDetailsSummaryViewSupported(),
       shouldAutoHorizontalScroll
     );
-    final TabInfo tabInfo = new TabInfo(inspectorPanel).setActions(toolbarGroup, ActionPlaces.TOOLBAR)
-      .append(displayName, SimpleTextAttributes.REGULAR_ATTRIBUTES)
-      .setSideComponent(overflowAction.getActionButton());
+    final TabInfo tabInfo = new TabInfo(inspectorPanel)
+      .append(displayName, SimpleTextAttributes.REGULAR_ATTRIBUTES);
     tabs.addTab(tabInfo);
     state.inspectorPanels.add(inspectorPanel);
     if (selectedTab) {
       tabs.select(tabInfo, false);
     }
   }
 
+  private void addDisabledTab(String displayName,
+                              JBRunnerTabs runnerTabs,
+                              FlutterApp app,
+                              DefaultActionGroup toolbarGroup) {
+    final JPanel panel = new JPanel(new BorderLayout());
+    final JBLabel label = new JBLabel("Widget info not available in profile mode", SwingConstants.CENTER);
+    label.setForeground(UIUtil.getLabelDisabledForeground());
+    panel.add(label, BorderLayout.CENTER);
+
+    final TabInfo tabInfo = new TabInfo(panel)
```

Comment:
idea: might as well make this placeholder tab an InspectorTabPanel just to normalize the UI?

Reply:
commented above